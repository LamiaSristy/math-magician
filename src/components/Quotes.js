/* eslint-disable max-len */
import React from 'react';
import quotesStyles from '../styles/quotes.module.css';

function Quotes() {
  return (
    <div className={quotesStyles['quote-container']}>
      <p className={quotesStyles.quote}>
        Mathematics is the queen of science, and arithmetic the queen of mathematics.
        <small>  â€“Carl Friedrich Gauss.</small>
      </p>
    </div>
  );
}

export default Quotes;
