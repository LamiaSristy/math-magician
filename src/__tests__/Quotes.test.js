import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import Quotes from '../components/Quotes';

describe('Quotes', () => {
  test('renders the quotes component correctly', () => {
    const quote = renderer.create(<Quotes />).toJSON();
    expect(quote).toMatchSnapshot();
  });
});