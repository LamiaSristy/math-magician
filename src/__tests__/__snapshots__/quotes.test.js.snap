// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes renders the quotes component correctly 1`] = `
<div
  className="quote-container"
>
  <p
    className="quote"
  >
    Mathematics is the queen of science, and arithmetic the queen of mathematics.
    <small>
        â€“Carl Friedrich Gauss.
    </small>
  </p>
</div>
`;
